docs/source/developers/deployment.rst

```diff
@@ -29,74 +29,81 @@ Once you have a repository ready:
   # Authenticate to quay.io
   docker login <credentials> quay.io
 
-  # Tag the build 
-  docker tag ibm-spectrum-scale-csi-operator quay.io/<your-user>/ibm-spectrum-scale-csi-operator:v0.9.1
+  # Configure some variables
+  VERSION="v0.9.1"
+  MYUSER="<your-user>"
 
-  # push the image
-  docker push quay.io/<your-user>/ibm-spectrum-scale-csi-operator:v0.9.1
+  # Tag and push the operator image 
+  docker tag ibm-spectrum-scale-csi-operator quay.io/${MYUSER}/ibm-spectrum-scale-csi-operator:${VERSION}
+  docker push quay.io/${MYUSER}/ibm-spectrum-scale-csi-operator:${VERSION}
 
-  # Update your deployment to point at your image.
-  hacks/change_deploy_image.py -i quay.io/<your-user>/ibm-spectrum-scale-csi-operator:v0.9.1
+  # Tag and push the driver image
+  docker tag ibm-spectrum-scale-csi-driver quay.io/${MYUSER}/ibm-spectrum-scale-csi-driver:${VERSION}
+  docker push quay.io/${MYUSER}/ibm-spectrum-scale-csi-driver:${VERSION}
+
+  # OPERATOR_DIR has been defined in previous steps
+  cd ${OPERATOR_DIR}
+  # Use a helper script to update your deployment to point at your operator image
+  hacks/change_deploy_image.py -i quay.io/${MYUSER}/ibm-spectrum-scale-csi-operator:${VERSION}
   
 
-Installing Operator
-```````````````````
+Installing the CSI Operator
+---------------------------
 
 .. note:: For OpenShift environments, replace ``kubectl`` with  ``oc``
 
-Run the following to deploy the operator manually:
+Run the following to deploy the IBM Spectrum Scale CSI operator manually:
 
 .. code-block:: bash
 
-  cd ${OPERATOR_DIR}/stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator
-
-  kubectl apply -f deploy/namespace.yaml
-  kubectl apply -f deploy/operator.yaml
-  kubectl apply -f deploy/role.yaml
-  kubectl apply -f deploy/role_binding.yaml
-  kubectl apply -f deploy/service_account.yaml
-  kubectl apply -f deploy/crds/ibm-spectrum-scale-csi-operator-crd.yaml
+  # OPERATOR_DIR has been defined in the previous steps
+  kubectl apply -f ${OPERATOR_DIR}/deploy/namespace.yaml
+  kubectl apply -f ${OPERATOR_DIR}/deploy/operator.yaml
+  kubectl apply -f ${OPERATOR_DIR}/deploy/role.yaml
+  kubectl apply -f ${OPERATOR_DIR}/deploy/role_binding.yaml
+  kubectl apply -f ${OPERATOR_DIR}/deploy/service_account.yaml
+  kubectl apply -f ${OPERATOR_DIR}/deploy/crds/ibm-spectrum-scale-csi-operator-crd.yaml
```

Comment:
Should we instruct the use of autogenerated file ibm-spectrum-scale-csi-operator.yaml ?

Reply:
Sure, let me make a change for this.

Reply:
On 2nd thought, this section is targeted for developers that may want contribute changes to the operator.... The changes to these yaml need to happen in those separate files, and the  generated file is created  from that.  

So asking the user to apply/delete that generated  file may cause confusion and would also fail the CI/CD if that is checked in, without the other files checked in.